src/sdk.js

```diff
@@ -504,6 +508,15 @@ const logoutInterceptors = [
   new FetchRefreshTokenForRevoke(),
 ];
 
+const exchangeTokenInterceptors = [
+  new FetchAuthTokenFromStore(),
```

Comment:
Should this also attempt to fetch the token from the API?

Reply:
@kpuputti By "current token" you mean the token for logged in user, not the anonymous token?

To get the current token for logged in user, you need either a) user's login credential or b) refresh token.

You already have the `RetryWithRefreshToken` interceptor there, so b) is covered.

For a), user should call `sdk.login` before attempting token exchange.

So nothing to add here if I understand the question right.